functions/dotnet/PowertoolsWorkshop.csproj

```diff
@@ -25,4 +25,7 @@
     <PackageReference Include="AWSSDK.S3" Version="3.7.308.8" />
     <PackageReference Include="SkiaSharp.NativeAssets.Linux.NoDependencies" Version="2.88.6" />
   </ItemGroup>
+  <ItemGroup>
+    <Compile Remove="module*/**/[complete]*.cs" />
```

Comment:
This line tells the compiler to ignore [complete] files, otherwise there would be compile errors on build

Reply:
Thank you! When we first created this last year I thought we couldn't have them side by side and had to create a separate folder, good that it's no the case and we can streamline the structure!